decorators/placeholder.js

```diff
@@ -6,18 +6,58 @@ var _ = require('lodash'),
   addComponentHandler = require('../services/components/add-component-handler');
 
 /**
+ *
+ * @param {Array} groupItems
+ * @param {string} propName
+ * @returns {object}
+ */
+function findPropInFieldGroup(groupItems, propName) {
+  return _.find(groupItems, groupProp => _.get(groupProp, '_schema._name') === propName) || null;
+}
+
+/**
+ * get the property value
+ * @param {string} path
+ * @param {object} data
+ * @param {string} propName
+ * @returns {String}
+ */
+function getPropVal(path, data, propName) {
+  var value = 'value';
+
+  return String( // always return a string
```

Comment:
Hmm, good point. How about this?

* write a comment above explaining that ("cast the value to a string. using a constructor will prevent errors being thrown" or something)
* use `new String()`